azure-pipelines/builds/ci.yml

```diff
@@ -29,10 +29,32 @@ stages:
         publish:
           artifacts: true
           manifests: true
+          logs: true
       enableSourceBuild: true
-  - template: /azure-pipelines/builds/generatescript-tests.yml
-    parameters:
-      buildConfiguration: Debug
+      sourceBuildParameters:
+        displayName: SourceBuild_Linux
+
+  ## The job below is a temporary workaround for running the tests on Windows.
+  ## Currently, the source-build-reference-packages repository does not support building on Windows, but it is capable of running the tests on Windows.
+  ## This job should be modified once the source-build-reference-packages repository supports building on Windows by transitioning to the template above.
+  - job: Test_Windows
+    strategy:
+      matrix:
+        Windows:
+          agentOs: Windows_NT
+          imageName: "windows-latest"
+    pool:
+      vmImage: $(imageName)
+    steps:
+    - checkout: self
+      clean: true
+    - script: $(Build.SourcesDirectory)/test.cmd
+      displayName: Run Tests
+    - task: PublishTestResults@2
```

Comment:
We don't want to run the publishing `always` like we do with the Linux job?

Reply:
Added in https://github.com/dotnet/source-build-reference-packages/pull/824/commits/b2930c5520b6e39c9e578184bbd8cef3186796db:

https://github.com/dotnet/source-build-reference-packages/blob/b2930c5520b6e39c9e578184bbd8cef3186796db/azure-pipelines/builds/ci.yml#L56